src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -230,7 +231,7 @@ function validateParams(vmParams) {
         validationFailed.memory = _("Memory must not be 0");
     }
 
-    if (vmParams.storagePool == 'NewVolume' && vmParams.storageSize === 0) {
+    if (vmParams.storagePool == 'NewVolumeQCOW2' && vmParams.storagePool == 'NewVolumeRAW' && vmParams.storageSize === 0) {
```

Comment:
The check is wrong here. Will be always false.

Reply:
Fixed